docs/FilesApi.md

```diff
@@ -114,7 +114,7 @@ Name | Type | Description  | Notes
 [[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
 
 # **get_files**
-> File get_files(id=id)
+> InlineResponse200 get_files(id=id)
```

Comment:
hmm i wonder why `File` got changed to `InlineResponse200` in this PR. nothing has changed in the spec for the files endpoint. did the openapi-generator version change or something? could you see if you can figure out why that changed?

Reply:
Hmmm, good catch. 

I noticed this commit in `front` that modified `File` related swagger docs:

https://github.com/lilt/front/commit/aff555ccad67eb28ffb9ab82a0e5cd6b096a42d0#diff-d30f236ae744363691f88a5a21da0bc0

I didn't consciously upgrade openapi-generator unless the node upgrade that front requires did it. Also found this explanation:

https://github.com/swagger-api/swagger-editor/issues/1319#issuecomment-300530614

I don't have the understanding to connect these dots, any thoughts @tgallant?

Reply:
Ah, good catch. I didn't know that had been updated. We may have to update the spec so it doesn't rename things to `InlineResponse200`. Let me take a look at that.

Reply:
Thanks!

Reply:
could you try generating it with this spec? i think adding names to the schemas should fix it https://github.com/lilt/front/tree/fix/adds-names-to-swagger-schema

Reply:
Hmm, still `InlineResponse200` in `FilesApi.md` 🤔

<img width="1792" alt="Screen Shot 2020-08-04 at 3 58 52 PM" src="https://user-images.githubusercontent.com/12013763/89353525-67d3cd00-d66b-11ea-99a4-3f677d8b4bc5.png">